### PR TITLE
Tree regression

### LINKED ISSUE #23: Implement DecisionTreeRegressor from scikit-learn library
Requires #26 
See #6

### LINKED ISSUE #67: Fix remove high std songs + bugfixes in storage/transformer
Fix remove high std songs + bugfixes in storage/transformer

### PR DESCRIPTION
**Pull request by tediiieee**
See commit msgs

Implemented tree regression

Also did:
refactors in ML
bugfixes in storage and transformer
Implemented new functions for accessing labels

Closes #23 
Closes #67

### CODE REVIEW

File: src/ml.py [hunk 1]
```diff
@@ -3,40 +3,42 @@
...
+                      in X. 2D numpy array with num_songs*time_samples rows.
     :return: Linear regressors for arousal and valence
     """
-    reg_arousal = LinearRegression().fit(X, y_arousal)
-    reg_valence = LinearRegression().fit(X, y_valence)
+    return LinearRegression().fit(X, y)
 
-    return(reg_arousal, reg_valence)
+
+def decision_tree(X, y):
```
Comment (sporreking) [inline src/ml.py:33]: Use correct docstring, even if other functions use incorrect format.

**Review by sporreking — CHANGES_REQUESTED** [review]
Use correct docstring in `decision_tree()`, even if other functions use incorrect format.

**Review by Ludvig-Joborn — APPROVED** [review]
Changes look great, good job!

**Review by sporreking — APPROVED** [review]
Looking good!

**Pull request merged**